feast_teradata/cli/template/teradata/bootstrap.py

```diff
@@ -17,7 +17,11 @@ def bootstrap():
     from feast.driver_test_data import create_driver_hourly_stats_df
 
     repo_path = pathlib.Path(__file__).parent.absolute()
-    project_name = str(repo_path)[str(repo_path).rfind("/") + 1 :]
+    if "/" in str(repo_path):
```

Comment:
this is confusing as you are using two different pieces of logic to get the project name from the string. One is `rfind` and the other is `split` and so different offsets/index is required in each. Also, what happens if `/` in windows path?

Anyway, I think something much simpler is to replace all this logic by the following. I'm not sure why feast had the more complex logic there to start with.

```python
project_name = os.path.basename(path)
```

Give it a test, it should work for you and its os independent.